orttraining/orttraining/eager/test/ort_ops.py

```diff
@@ -272,6 +272,27 @@ def test_masked_select_broadcast(self):
         assert torch.allclose(cpu_result, ort_result.cpu())
         assert cpu_result.dim() == ort_result.dim()
 
+    def test_bitwise_and(self):
+        device = self.get_device()
+        cpu_a = torch.tensor([[0], [1], [1]], dtype=bool)
+        cpu_b = torch.tensor([[1], [0], [1]], dtype=bool)
+        ort_a = cpu_a.to(device)
+        ort_b = cpu_b.to(device)
+        cpu_result = torch.bitwise_and(cpu_a, cpu_b)
+        ort_result = torch.bitwise_and(ort_a, ort_b)
+        assert torch.equal(cpu_result, ort_result.cpu())
+
+    def test_bitwise_and_fallback(self):
+        device = self.get_device()
+        # use randint because bitwise_and is not supported on floats
+        cpu_a = torch.randint(200, (3, 4))
+        cpu_b = torch.randint(200, (3, 4))
+        ort_a = cpu_a.to(device)
+        ort_b = cpu_b.to(device)
+        cpu_result = torch.bitwise_and(cpu_a, cpu_b)
+        ort_result = torch.bitwise_and(ort_a, ort_b)
```

Comment:
i recently discovered OutputGrabber for checking output (used in orttraining/orttraining/eager/test/ort_eps_test.py). We don't need to use it hear but something to consider if we add output about falling back to verify behavior.

Reply:
Cool, thanks!